src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -779,13 +847,26 @@ export class SearchView extends ViewPane {
 		e.browserEvent.preventDefault();
 		e.browserEvent.stopPropagation();
 
+		const match = e.element;
+		let context: IRenderableMatchContext | undefined = undefined;
+		if (match instanceof FileMatch) {
+			context = toFileMatchContext(match);
+		} else if (match instanceof Match) {
+			context = toMatchContext(match);
+		} else if (match instanceof FolderMatchWithResource) {
+			context = toFolderMatchContext(match);
+			context.resource = match.resource.toString();
```

Comment:
Why isn't this line in `toFolderMatchContext`?

Reply:
Fixed